### PR TITLE
Fix lock release logs

### PR DESCRIPTION
**Pull request by tyuchn**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
/kind bug

**What this PR does / why we need it**:
Fix lock release logs

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
N/A
```

### CODE REVIEW

File: pkg/csi_driver/node.go [hunk 1]
```diff
@@ -542,7 +542,7 @@ func (s *nodeServer) nodeStageVolumeUpdateLockInfo(ctx context.Context, req *csi
 	}
 
 	if err := s.lockReleaseController.UpdateConfigMapWithKeyValue(ctx, cm, lockInfoKey, filestoreIP); err != nil {
-		klog.Errorf("NodeStageVolume failed to update configmap %s/%s with lock info {%s: %s} for volume %s: %v", util.ManagedFilestoreCSINamespace, configmapName, volumeID, err)
+		klog.Errorf("NodeStageVolume failed to update configmap %s/%s with lock info {%s: %s} for volume %s: %v", util.ManagedFilestoreCSINamespace, configmapName, lockInfoKey, filestoreIP, volumeID, err)
```
Comment (msau42) [inline pkg/csi_driver/node.go:545]: You can use klog.KObj to simplify the name/namespace: https://github.com/kubernetes/klog/blob/6ded8085b27eeaf68a13d583a382b125ef1482ee/k8s_references.go#LL78C7-L78C7

Comment (tyuchn) [reply pkg/csi_driver/node.go]: Done. Thanks for the recommendation.

**Pull request merged**